x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/flyout/fields_table.tsx

```diff
@@ -44,35 +55,54 @@ export const IndicatorFieldsTable: FC<IndicatorFieldsTableProps> = ({
               defaultMessage="Value"
             />
           ),
-          render: (field: string) => <IndicatorFieldValue indicator={indicator} field={field} />,
+          render: (item: TableItem) => (
+            <SecurityCellActions
+              data={{ field: item.key, value: item.value }}
+              mode={CellActionsMode.HOVER_DOWN}
+              triggerId={SecurityCellActionsTrigger.DEFAULT}
+            >
+              <IndicatorFieldValue indicator={indicator} field={item.key} />
+            </SecurityCellActions>
+          ),
         },
+      ] as Array<EuiBasicTableColumn<TableItem>>,
+    [indicator]
+  );
+
+  const search = useMemo(
+    () => ({
+      box: {
+        schema: true,
+        incremental: true,
+      },
+    }),
+    []
+  );
```

Comment:
Extract this to a constant outside the component instead of using `useMemo` here